sdk/examples/xo_python/sawtooth_xo/xo_client.py

```diff
@@ -80,6 +82,28 @@ def show(self, name):
         except BaseException:
             return None
 
+    def _get_statuses(self, batch_ids):
+        try:
+            url = "http://{}/{}".format(self._base_url, 'batch_status')
+
+            if isinstance(batch_ids, bytes):
+                headers = {'Content-Type': 'application/octet-stream'}
+            else:
+                batch_ids = json.dumps(batch_ids).encode()
+                headers = {'Content-Type': 'application/json'}
+            headers['Content-Length'] = '%d' % len(batch_ids)
+
+            result = requests.post(url, data=batch_ids, headers=headers)
```

Comment:
If you are only every checking the status of one batch, this would be better formatted as a `GET /batch_status?id={batch id}&wait={wait time}`. The option to `POST /batch_status` is only there as a workaround to allow checking on any number of ids.  Since URLs are limited to ~2048 characters, and our ids are 128 characters long, you won't be able to check on more than ~15 ids with a `GET` request. If you don't need to check on more than that though, a `POST` is overkill.